spec/features/idv/doc_auth/redo_document_capture_spec.rb

```diff
@@ -11,418 +11,7 @@
-        )
-      end
-    end
-
-    context 'when doc auth and portrait match fail', allow_browser_log: true do
-      before do
-        allow(IdentityConfig.store).to receive(:use_vot_in_sp_requests).and_return(true)
-        allow_any_instance_of(DocAuth::Response).to receive(:selfie_status).and_return(:fail)
-        start_idv_from_sp(facial_match_required: true)
-        sign_in_and_2fa_user
-        complete_doc_auth_steps_before_document_capture_step
-        mock_doc_auth_fail_face_match_fail
-        attach_images
-        attach_selfie
-        submit_images
-        click_try_again
-        sleep(10)
-      end
-
-      it_behaves_like 'document and selfie images re-upload not allowed'
-    end
-
-    context 'when pii validation fails', allow_browser_log: true do
-      before do
-        allow(IdentityConfig.store).to receive(:use_vot_in_sp_requests).and_return(true)
-        pii = Idp::Constants::MOCK_IDV_APPLICANT.dup
-        pii[:address1] = nil
-        allow_any_instance_of(DocAuth::LexisNexis::Responses::TrueIdResponse).
-          to receive(:pii_from_doc).and_return(Pii::StateId.new(**pii))
-        start_idv_from_sp(facial_match_required: true)
-        sign_in_and_2fa_user
-        complete_doc_auth_steps_before_document_capture_step
-        mock_doc_auth_pass_face_match_pass_no_address1
-        attach_images
-        attach_selfie
-        submit_images
-        click_try_again
-        sleep(10)
-      end
-
-      it 'shows selfie inline error messages for both front and back' do
-        expect(page).to have_css(
-          '.usa-error-message[role="alert"]',
-          text: t('doc_auth.errors.general.multiple_front_id_failures'),
-          count: 1,
-        )
-        expect(page).to have_css(
-          '.usa-error-message[role="alert"]',
-          text: t('doc_auth.errors.general.multiple_back_id_failures'),
-          count: 1,
-        )
-      end
-
-      it 'stops user submitting the same images again' do
-        expect(fake_analytics).to have_logged_event('IdV: doc auth document_capture visited')
-        expect(fake_analytics).to have_logged_event(
-          'IdV: doc auth image upload form submitted',
-          hash_including(remaining_submit_attempts: 3, submit_attempts: 1),
-        )
-        DocAuth::Mock::DocAuthMockClient.reset!
-        expect(page).not_to have_css(
-          '.usa-error-message[role="alert"]',
-          text: t('doc_auth.errors.doc.resubmit_failed_image'),
-        )
-
-        attach_selfie
-        expect(page).not_to have_css(
-          '.usa-error-message[role="alert"]',
-          text: t('doc_auth.errors.doc.resubmit_failed_image'),
-        )
-
-        attach_images
-        expect(page).to have_css(
-          '.usa-error-message[role="alert"]',
-          text: t('doc_auth.errors.doc.resubmit_failed_image'),
-          count: 2,
-        )
-      end
-    end
-  end
-
   context 'split doc auth flow', allow_browser_log: true do
-    before do
```

Comment:
the `context 'split doc auth flow'` is no longer necessary since there is no alternative